joomla/joomla-cms
# [4.4] Backendmenu: Fixing export of additionla features for menu items

Partial Pull Request for Issue #41268 .

### Summary of Changes
While looking at #41268 I noticed that the export of backend menus currently is not properly exporting the attributes for the quicktask, dashboard and ajax-badge feature. This PR fixes that.

Please note that the export also currently does not properly export the tree structure of the menu. This is fixed in #43178.

Please also note, that the backend menu items don't properly support the quicktask, dashboard and ajax badge features in the menu item configuration yet. This is something that needs to be fixed in another PR.


### Testing Instructions
1. Go to menu manager, filter by admin and create a new menu for the administrator. Make sure to select a preset to pre-populate the menu.
2. In the menu manager, select the menu and click "Download as Preset"
3. Compare the preset with your download


### Actual result BEFORE applying this Pull Request
There is no `dashboard=""`, `quicktask=""`, `"ajax-badge=""` or icon, title and permission attribute for the quicktask in the XML. The preset and the export differ wildly.


### Expected result AFTER applying this Pull Request
There are attributes for the above mentioned features and the preset and export are a lot more similar.


### Link to documentations
Please select:
